shared/constants.js

```diff
@@ -25,92 +25,233 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+var path = require('path'),
+    shelljs = require('shelljs');
+
 var VERSION = '6.0.0';
 
 module.exports = {
     version: VERSION,
 
+    tools: {
+        git: {
+            checkCmd: 'git --version',
+            minVersion: '2.13'
+        },
+        npm: {
+            checkCmd: 'npm -v',
+            minVersion: '3.10'
+        },
+        pod: {
+            checkCmd: 'pod --version',
+            minVersion: '1.2'
+        },
+        cordova: {
+            checkCmd: 'cordova -v',
+            minVersion: '7.0.0',
+            pluginRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin#dev',    // dev
+            //pluginRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin#v' + VERSION, // GA
+            platformVersion: {
+                ios: '4.4.0',
+                android: '6.2.3'
+            }
+        }
+    },
+
+    ides: {
+        ios: 'XCode',
+        android: 'Android Studio'
+    },
+
     templatesRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#dev',    // dev
     //templatesRepoUri: 'https://github.com/forcedotcom/SalesforceMobileSDK-Templates#v' + VERSION, // GA
 
 
     forceclis: {
         forceios: {
             name: 'forceios',
-            description: 'command line utility for building iOS native mobile applications using Salesforce Mobile SDK',
```

Comment:
There is a one to one mapping between our force* clis and the sfdx plugin topics.
If one day, it is no longer the case, I'll define a separate map with sfdx topics.